packages/client/components/MenuToggleV2Text.tsx

```diff
@@ -37,14 +33,14 @@ const MenuToggleTitle = styled('div')({
 
 interface Props {
   label: string
-  icon?: string
+  icon?: React.ReactElement
   title: string
 }
 const MenuToggleV2Text = forwardRef((props: Props, ref: any) => {
```

Comment:
+1 Maybe we could find a better name for this component?

Reply:
Renamed to `startAdornment`, taking inspiration from material-ui library

Reply:
`icon` name was fine, I commented on the `MenuToggleV2Text` name. `startAdornment` is ok, but slightly less intuitive.

Reply:
Renamed back to icon, renamed MenuToggleV2Text